packages/datagateway-common/src/homePage/homePage.component.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import {
+  withStyles,
+  Theme,
+  WithStyles,
+  Grid,
+  createStyles,
+} from '@material-ui/core';
+import { StyleRules } from '@material-ui/core/styles';
+import Image from '../images/Jasmin4 _DSC7054.jpg';
+import DatagatewayLogoWithText from '../images/datagateway-logo-white.svg';
+import ExploreImage from '../images/explore.jpg';
+import DiscoverImage from '../images/discover.jpg';
+import DownloadImage from '../images/download.jpg';
```

Comment:
As shown in the screenshot, these images apart from `explore.jpg` are not found. I can see them in the `images` directory though.
![image](https://user-images.githubusercontent.com/45173816/115403626-815e7a00-a1e4-11eb-9fa0-bd70bc33fd43.png)

Reply:
Yeah I wasn't sure if it was just me. Okay thanks for that feedback, clearly something is going on there

Reply:
This has, finally, been fixed!